src/OrchardCore.Modules/OrchardCore.Microsoft.Authentication/Startup.cs

```diff
@@ -49,6 +51,7 @@ public class AzureADStartup : StartupBase
     {
         public override void ConfigureServices(IServiceCollection services)
         {
+            services.AddRecipeExecutionStep<AzureADSettingsStep>();
```

Comment:
Move this underneath `IAzureADService` registration, otherwise a runtime error will occur